app/views/declaration/declaration_choice.scala.html

```diff
@@ -91,16 +91,15 @@
                     checked = form("type").value.contains(DeclarationType.OCCASIONAL.toString)
                 ),
                 RadioItem(
-                    divider = Some(messages("declaration.type.or")),
-                    value = Some(RadioOption.dividerId)
+                    divider = Some(messages("declaration.type.or"))
```

Comment:
This change and line 102 has been changed by @SteveSugden  last time. Not sure if this change with removing value is correct

Reply:
No.  This looks like it's reverting my changes of this morning.